deploy/kube-config/google/heapster-service.json

```diff
@@ -1,23 +0,0 @@
-{
```

Comment:
While you are here, should we symlink the service definitions instead of duplicating them?